packages/client/src/api.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+import { resolveSelect } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { store } from './store';
+import type { Ability, AbilityInput, AbilityOutput } from './types';
+
+/**
+ * List all available abilities.
+ *
+ * @return Promise resolving to array of abilities.
+ */
+export async function listAbilities(): Promise< Ability[] > {
+	return await resolveSelect( store ).getAbilities();
+}
```

Comment:
Maybe we should use `getAbilities` for the API name as well. It will make it easier to remember.

Reply:
Agreed, renamed to getAbilities in https://github.com/WordPress/abilities-api/pull/60/commits/43ed397712f1b9028c3a29f67e51d4b507801b46